operations/messaging-and-math/content-marketing/video-content.md

```diff
@@ -0,0 +1,86 @@
+---
+description: '50% DRAFT (revised by Ashley Dotterweich on Oct 20, 2020)'
+---
+
+# Video Content Best Practices
+
+This document provides guidelines for creating video content for Mattermost. It covers:
+
+* What is video content used for?
+* How can I create great video content?
+* What is the process for developing video content?
+* What is the process for publishing video content?
+* How is video content promoted?
+
+## The Pros and Cons of Video Content
```

Comment:
I recommend removing this as the subheadings don't address the heading.

We could then break down the subheadings into H3 headers so they're easier to navigate and appear in the TOC. @aedott, @amyblais - thoughts?

Reply:
Makes sense to me!